### ISSUE TITLE
Ability to only trigger binding when keys held down in specific order

### ISSUE THREAD

**Issue by waddyhussain** [sezanzeb/input-remapper#826]
Not sure if this is already available, but is it possible to have a feature where you can set a binding to only trigger when keys are held down in a specific order? (or maybe the reverse, i.e binding is not triggered when keys held down in a specific order, but in all other cases it is triggered, personally I don't mind on the exact implementation)

My use case is I have BTN_SIDE + BTN_MIDDLE on my mouse mapped to close the current window. Sometimes when I'm holding down the scroll wheel I'll accidentally click the side button and close a window. I'd like set it so it only triggers if I hold down BTN_SIDE first. 

Perhaps it doesn't have to be as complex as defining a whole sequence, but just marking specific keys that should be held down first, and then the remaining keys come after.

**Comment by htgar** [comment]
Perhaps you could try defining multiple inputs under your preset? One for BTN_SIDE + BTN_MIDDLE and one for BTN_MIDDLE + BTN_SIDE. From what I can tell, this follows the order the keys are pressed.

What program are you using to map your mouse keys to close the current window?

**Comment by waddyhussain** [comment]
Tried your suggestion and it works, thanks a lot for the help! Set BTN_MIDDLE + BTN_SIDE to BTN_MIDDLE with release inputs disabled, and now it only closes the window when the side button is pressed first.

In input remapper I just have the mouse combo remapped to Super + Q, which is the keybind I have set in my window manager to close the current window, so I'm not using any specific program for that.
